icicle/appUtils/ntt/tests/verification.cu

```diff
@@ -19,29 +19,65 @@ typedef curve_config::scalar_t test_data;
 void random_samples(test_data* res, uint32_t count)
 {
   for (int i = 0; i < count; i++)
+    // res[i] = i==0 ? test_scalar::one() : test_scalar::zero();
```

Comment:
remove commented code. Also in other places

Reply:
done